ROADMAP.md

```diff
@@ -12,56 +12,57 @@ We use the [milestone](https://github.com/rook/rook/milestones) feature in Githu
 
 * Update project governance policies [#1445](https://github.com/rook/rook/issues/1445)
 * Add Core Infrastructure Initiative (CII) Best Practices [#1440](https://github.com/rook/rook/issues/1440)
-* Integrate a more robust controller framework (e.g., CoreOS Operator SDK or Kubebuilder) [#1981](https://github.com/rook/rook/issues/1981)
```

Comment:
what do you think of a more robust operator/controller framework for v1.0?

Reply:
Before it's added to the roadmap for 1.0, I'd recommend some investigation to the frameworks. It's not clear (at least to me) what the benefits are to move to a new framework. Until that is understood, other features will take priority. I'm not against it, I'm just resisting moving everything by default to 1.0.